src/assets/locales/en.json

```diff
@@ -734,6 +734,7 @@
         "damage": "{0} suffer %game.damage:{2}%",
         "gainCondition": "{0} gain %game.condition.{1}%",
         "hp": "{0} HP to {2}",
+        "discard": "{0} discards {2} cards",
```

Comment:
can you change `{2}` to `%number.{2}%`? I will add number label for translation.